httpclient_test.go

```diff
@@ -121,8 +121,6 @@ func TestGetJSON_PanicOnNilResponse(t *testing.T) {
 
 	_, _, err := GetJSON(nil, resp, errResp, "GET", ":", "", nil)
 
-	if err == nil {
-		t.Fatal("err should not be nil")
-	}
+	should.NotBeNil(t, err)
```

Comment:
my previous suggestion was bad 
```suggestion
	should.BeError(t, err)
```